Lib/ldap/controls/deref.py

```diff
@@ -108,7 +108,7 @@ def decodeControlValue(self,encodedControlValue):
     for deref_res in decodedValue:
       deref_attr,deref_val,deref_vals = deref_res[0],deref_res[1],deref_res[2]
       partial_attrs_dict = {
-        str(tv[0]): map(str,tv[1])
+        str(tv[0]): [str(v) for v in tv[1]]
```

Comment:
In Python 2, list comprehensions leak the internal variable. Therefore I prefer list() for list comprehensions.

Reply:
I think it's OK to use comprehensions with one-letter variable names.
No need to change this.